eth/filters/api.go

```diff
@@ -348,7 +349,11 @@ func (api *FilterAPI) GetLogs(ctx context.Context, crit FilterCriteria) ([]*type
 			end = crit.ToBlock.Int64()
 		}
 		// Construct the range filter
-		filter = api.sys.NewRangeFilter(begin, end, crit.Addresses, crit.Topics)
```

Comment:
I agree with return an error in this case. But we should raise it on the execution-apis repo for discussion and specify it there first.

Reply:
ref https://github.com/ethereum/execution-apis/pull/480